lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -1103,7 +1103,7 @@ def test_static_badlink_rewrite(self):
         result_fragment = module.render(STUDENT_VIEW)
 
         self.assertIn(
-            '/c4x/{org}/{course}/asset/_file.jpg'.format(
+            '/c4x/{org}/{course}/asset/file.jpg'.format(
```

Comment:
Good to know both will continue to work, although in practice folks at MIT typically use the `subdir/file.png` construct. (I suspect because it maps better to their mental model of the course structure)

Here's an example from [6.004.2x](https://courses.edx.org/courses/course-v1:MITx+6.004.2x+3T2015/courseware/c3/c3s1/1?activate_block_id=block-v1%3AMITx%2B6.004.2x%2B3T2015%2Btype%40vertical%2Bblock%40c3s1v1)  (thanks @sfrucht for digging this up!)

```
<script src="/static/video_fix.js"></script>
<a href="/static/pdfs/isa/c9s1v1.pdf" target="_blank">Slides (PDF) for this lesson.</a>
```